HW/Oleksandr Liashenko/HW06_Task1.py

```diff
@@ -0,0 +1,18 @@
+# A set for even numbers that are divisible by 2
+even_div2 = set()
+# A set for odd numbers that are divisible by 3
+odd_div3 = set()
+# A set for all numbers in range
+full_range = set()
+# A loop to determine numbers and fill our sets
+for i in range(1, 11, 1):
+    full_range.add(i)
+    if i % 2 == 0:
+        even_div2.add(i)
+    if i % 3 == 0:
+        odd_div3.add(i)
```

Comment:
some even can be here

Reply:
Agree, my mistake. I should change "if i % 3 == 0" for "elif i % 3 == 0"